backend/src/openbeheer/utils/tests/__init__.py

```diff
@@ -1,12 +1,59 @@
+from typing import Callable, Iterable
+
 from django.test import TestCase as _TestCase, tag
 
 from maykin_common.vcr import VCRMixin as _VCRMixin
 from rest_framework.test import APITestCase as _APITestCase
 from vcr.cassette import Cassette
+from vcr.request import Request
+
+
+def matcher_query_without_datum_geldigheid(
+    incoming_request: Request, stored_request: Request
+) -> None:
+    """Match requests ignoring query param datumGeldigheid when retrieving informatieobjecttypen."""
+
+    incoming_query_names = {query_name for query_name, _ in incoming_request.query}
+    stored_query_names = {query_name for query_name, _ in stored_request.query}
+    assert incoming_query_names == stored_query_names, (
+        "The names of the query params in the incoming request don't match those in the stored request."
+    )
+
+    if (
+        incoming_request.path == "/catalogi/api/v1/informatieobjecttypen"
+        and "datumGeldigheid" in stored_query_names
```

Comment:
```suggestion
        "datumGeldigheid" in stored_query_names
```
Or no? Looks like a useful matcher to have for other endpoints too... :thinking: Or will this become too broad and match bogus responses.

Reply:
I hesitated :thinking: Initially I had it for all endpoints, but then I was worried that it would be too broad and changed it :sweat_smile: